.cirrus.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 task:
   install_script: pkg install -y bash protobuf go git python3 && python3 -m ensurepip
-  build_script: PLZ_ARGS="-o cache.httpurl:http://$CIRRUS_HTTP_CACHE_HOST" ./bootstrap.sh --exclude pip --exclude py2 --exclude no_cirrus --exclude cc
+  build_script: ./bootstrap.sh --exclude pip --exclude py2 --exclude no_cirrus
```

Comment:
It would be nice to get some testing of the http cache. Maybe we can write some integration tests though.

Reply:
Yep agreed; I'm not sure if this will be a permanent thing. Ideally we'd be able to re-add it since it would make Cirrus builds faster if it were working again.